### PR TITLE
some fixes

### PR DESCRIPTION
**Pull request by TheDovahkiin97**

### CODE REVIEW

**Comment by TheDovahkiin97** [comment]
the structure is orientated at the new and very detailed localisations for magyar cultures.
but the ``job_spiritual_``, ``temple_`` and ``title_`` cultural localisations dont work at all because of hardcoding. They are still featured because i believe in miracles x)

i will try to adjust bulgarian and bolghar someday

**Comment by TheDovahkiin97** [comment]
adding the same cultural localisations for all game-languages is not needed as "english" as the first one is the default fallback

**Comment by TheDovahkiin97** [comment]
@hmlendea i have seen your commit and you made some mistakes while cutting supposedly redundant localisations. I cuted those you were right with in my last commit here but the others are definitely needed.
Also some other minor fixes like a the b_zdrelo holding they added.
This branch is safe to merge regardless what travis says and i explained above why long ago

**Comment by TheDovahkiin97** [comment]
if you dont want to merge the entire thing for some reason at least copy the main file and merge it

**Comment by hmlendea** [comment]
In the previous commit i removed some titles that were not redundant but they had invalid characters (?) and I couldn't find what those characters should actually be. Can you take a look, and if you find anything, add them back?

File: hip-more-cultural-names/common/landed_titles/0_HIP_MoreCulturalNames.txt [hunk 1]
```diff
@@ -22542,18 +22525,18 @@ e_poland = {
             }
         }
         d_mazovia = {
-            bosnian = "Mazovi?a"
-            bulgarian = "Mazovi?a"
-            carantanian = "Mazovi?a"
-            croatian = "Mazovi?a"
+            bosnian = "Mazovija"
+            bulgarian = "Mazovija"
```
Comment (TheDovahkiin97) [inline hip-more-cultural-names/common/landed_titles/0_HIP_MoreCulturalNames.txt]: i guess you mean these i already fixed in this branch long ago

**Comment by hmlendea** [comment]
Closing due to completely replacing all the files with a brand new version (#25 )

**Pull request closed without merging**